addons/beehave/nodes/beehave_root.gd

```diff
@@ -32,18 +32,8 @@ func _ready() -> void:
 
 func _physics_process(delta: float) -> void:
 	blackboard.set_value("delta", delta)
-
 	var status = self.get_child(0).tick(actor, blackboard)
 
-	if status != RUNNING:
-		blackboard.set_value("running_action", null)
-
-
-func get_running_action() -> ActionLeaf:
```

Comment:
This method cannot be removed, as it is used actively. For example, I depend on it in my game to display debug information about the tree. To avoid breaking changes, you will need to find a way to re-introduce this.

Reply:
Oh ok, I could not find a reference thats why I removed. But sure, it gets hard to debug without this. I will add it back.